Packs/MicrosoftTeams/ReleaseNotes/1_4_62.json

```
@@ -0,0 +1,4 @@
+{
+  "breakingChanges":true,
+  "breakingChangesNotes":"Updated the parameter 'Minimum incident severity to send notifications to Teams by', since it is not supported by XSIAM."
```

```suggestion
  "breakingChangesNotes":"Hid the `Minimum incident severity to send notifications to Teams by` parameter, since it is not supported on XSIAM."
```
Please change in all release notes too